neo4j-to-neptune/src/main/java/com/amazonaws/services/neptune/util/NeptuneBulkLoader.java

```diff
@@ -105,12 +114,12 @@ public NeptuneBulkLoader(BulkLoadConfig bulkLoadConfig) {
                 .credentialsProvider(DefaultCredentialsProvider.create())
                 .multipartEnabled(true)
                 .httpClientBuilder(NettyNioAsyncHttpClient.builder()
-                    .maxConcurrency(200) // Increase max concurrent connections further
-                    .maxPendingConnectionAcquires(2000) // Increase pending connection queue
-                    .connectionAcquisitionTimeout(Duration.ofMinutes(10)) // 10 minute timeout
-                    .connectionTimeout(Duration.ofMinutes(5)) // 5 minute connection timeout
-                    .readTimeout(Duration.ofMinutes(15)) // 15 minute read timeout
-                    .writeTimeout(Duration.ofMinutes(15)) // 15 minute write timeout
+                    .maxConcurrency(300)
+                    .maxPendingConnectionAcquires(5000)
+                    .connectionAcquisitionTimeout(Duration.ofHours(3))
+                    .connectionTimeout(Duration.ofMinutes(5))
+                    .readTimeout(Duration.ofMinutes(15))
+                    .writeTimeout(Duration.ofMinutes(15))
```

Comment:
It seems like quite a challenge to properly tune all of these timeouts. We were able to get away without configuring all of these timeouts in Neptune Export by using the `AwsCrtAsyncHttpClient` instead of `NettyNioAsyncHttpClient`. https://github.com/aws/neptune-export/blob/5b7349c67cd2e95b702d14be3bb8f6c5d3624141/src/main/java/com/amazonaws/services/neptune/util/TransferManagerWrapper.java#L33-L35

I wonder if a similar approach would be viable here. The data transfer requirements should be comparable between the projects. Amazon has a doc for selecting the appropriate client, the most notable difference is that the Crt client does not support HTTP2. https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/http-configuration.html